Expensify/App
# See extra fields per chat

@stitesExpensify, please review
CC: @yuwenmemon 

### Details
This sets up information used in default room type chats.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/161776

### Tests
Tested that this change lets you load reports and doesn't cause the app to crash, since we don't have any front end changes for this feature yet.

### QA Steps
1. Logged into an account
2. Opened the chrome dev tools with Cmd + Option + J and went to the Application tab then local storage
3. Filtering for `report_` go through a few of the options and verify that ownerEmail, policyID, and chatType are there in the object (see screenshot for reference)

### Tested On


### Screenshots
Essentially just checking that the values: ownerEmail, policyID, and chatType  are set in localstorages. They'll be used in a later PR.

#### Web, Mweb, and Desktop
![image](https://user-images.githubusercontent.com/19364431/121246913-a3cd5500-c86f-11eb-9815-2cfcfad47413.png)
